website/docs/r/log_analytics_data_export_rule.html.markdown

```diff
@@ -60,6 +60,14 @@ The following arguments are supported:
 
 * `enabled` - (Optional) Is this Log Analytics Data Export Rule enabled? Possible values include `true` or `false`. Defaults to `false`.
 
+* `metadata` - (Optional) A `metadata` block as defined below.
+
+---
+
+The `metadata` block supports the following attributes:
```

Comment:
is this metadata? or destinations? should be bother putting this inside a block?

Reply:
Closed this comment as duplicate